### PR TITLE
Fix account monitor page

### LINKED ISSUE #5310: Account monitoring page does not show available balance
### Expected behavior
The account monitoring page should show the list of token balances available to select from

### Actual behavior
Account monitoring page does not show the list of token balance available on the current account

### Steps to reproduce
- Navigate to the account monitor page
- Tigger the token select dropdown field

### PR DESCRIPTION
**Pull request by eniolam1000752**
### What was the problem?

This PR resolves #5310

### How was it solved?

- [x] Get tokens from available not tokens metadata
- [x]  Fix failing unit tests

### How was it tested?
- Navigate to the account monitor page
- You should be able to see the list of tokens available on the account which can be selected to get the top accounts.

### CODE REVIEW

File: src/modules/account/components/Accounts/Accounts.js [hunk 1]
```diff
@@ -1,33 +1,30 @@
 import React, { useEffect } from 'react';
-
```
Comment (oskarleonard) [inline src/modules/account/components/Accounts/Accounts.js:2]: NRTTL:
My account has bot LSK and ENVT. 

When i am on LSK chain
image

When i am on Enevti i see both
image

Is this intended behaviour? Shouldnt i see both in each case?

**Review by ikem-legend — DISMISSED** [review]

**Review by ManuGowda — DISMISSED** [review]

**Review by ManuGowda — DISMISSED** [review]

File: src/modules/token/fungible/hooks/useTransferableTokens.test.js [hunk 2]
```diff
@@ -1,20 +1,28 @@
 import { renderHook } from '@testing-library/react-hooks';
 import mockSavedAccounts from '@tests/fixtures/accounts';
-import * as queries from '@token/fungible/hooks/queries';
+// import * as queries from '@token/fungible/hooks/queries';
```
Comment (oskarleonard) [inline src/modules/token/fungible/hooks/useTransferableTokens.test.js]: comment

Comment (eniolam1000752) [inline src/modules/token/fungible/hooks/useTransferableTokens.test.js]:
```suggestion
```

**Review by ManuGowda — DISMISSED** [review]

File: src/modules/account/components/Overview/overview.js [hunk 3]
```diff
@@ -44,6 +39,7 @@ const Overview = ({ tokenData, setFilter }) => {
           select={(selectedValue, option) => selectedValue?.tokenName === option.tokenName}
           className={styles.menuWrapper}
           popupClassName={styles.popupWrapper}
+          disabled={isFetching}
         >
           {tokens.map((tokenInfo) => (
```
Comment (oskarleonard) [inline src/modules/account/components/Overview/overview.js:43]: NRTTL
We should probably hide this wrapper if no items? Like when on Betanet 🤔 . But shouldnt betanet show something here 🤔 

image

File: src/modules/account/components/Overview/overview.js [hunk 4]
```diff
@@ -1,21 +1,17 @@
-import React, { useState, useRef, useEffect } from 'react';
+/* eslint-disable max-statements */
```
Comment (oskarleonard) [inline src/modules/account/components/Overview/overview.js]: Can we move these just above the head of the component keeping the import section clean

File: src/modules/token/fungible/hooks/queries/useNetworkSupportedTokens.js [hunk 5]
```diff
@@ -1,4 +1,5 @@
-import { useMemo, useRef } from 'react';
+/* eslint-disable max-statements */
```
Comment (oskarleonard) [inline src/modules/token/fungible/hooks/queries/useNetworkSupportedTokens.js]: move down above useNetworkSupportedTokens

File: src/locales/en/common.json [hunk 6]
```diff
@@ -663,6 +663,7 @@
   "The validator is temporarily punished and their validator weight is set to 0 due to a misbehavior.": "The validator is temporarily punished and their validator weight is set to 0 due to a misbehavior.",
   "The validator weight is below 1,000 LSK meaning that the validator is not eligible to generate.": "The validator weight is below 1,000 LSK meaning that the validator is not eligible to generate.",
   "The {{inputLabel}} can't be empty": "The {{inputLabel}} can't be empty",
+  "There are no accounts to display.": "There are no accounts to display.",
```
Comment (oskarleonard) [inline src/locales/en/common.json:666]: NRTTL:
Dont know if this i BE or FE. But isnt it strange that being on lisk application showing +20 accounts but then 
image

switching to enevti and filter by lisk and now it only shows 7 accounts
image

Also being on lisk sorting by enevti gives no results
image

And then being on Enevti we suddenly have accounts again
image

**Review by oskarleonard — APPROVED** [review]

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**